readme.md

```diff
@@ -140,7 +140,3 @@ All fields in this list can be used together with the computed fields plugin. Th
 ## Contributing
 
 See [contributing.md](https://github.com/voorhoede/datocms-plugin-computed-fields/blob/main/contributing.md).
```

Comment:
Think I would remove this part personally since GH already shows a popup to people that open a new issue or PR with a link to this file. Curious what you think of this.

Reply:
Since the readme is also the exact page that DatoCMS shows (https://www.datocms.com/marketplace/plugins/i/datocms-plugin-computed-fields) I think it's nice to have

Reply:
Let's merge this and see if the readme on the datocms plugin page updates or if it's using the npm readme. Can't find any datocms plugin repo with a lowercase readme file.

Reply:
358f24b